part-spec/power.json

```diff
@@ -697,6 +697,515 @@
+          "$ref": "definitions.json#/unit"
+        },
+        "vinMax": {
+          "description": "maximum input voltage under which the part can be expected to reliabily operate ",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswTyp": {
+          "description": "typical switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMin": {
+          "description": "minimum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMax": {
+          "description": "maximum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentConsumption": {
+          "description": "current consumption of a device",
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/currentConsumption"
+          }
+        },
+        "componentProtectionThresholds": {
+          "description": "Thermal and power supply protection thresholds of a device",
+          "type": "array",
+          "items": {
+            "$ref": "icProtection.json#/componentProtectionThresholds"
+          }
+        },
+        "integratedLoadSwitch": {
+          "description": "whether the device contains integrated power path load switch(es)",
+          "type": "boolean"
+        },
+        "integratedFets": {
+          "description": "whether the device contains integrated switching mosfets",
+          "type": "boolean"
+        },
+        "integratedFetProperties": {
+          "description": "describes integrated fet current limits and rdson properties",
+          "$ref": "#/$defs/powerFetProperities"
+        },
+        "bc12Support": {
+          "description": "whether bc 1.2 detection is built in",
+          "type": "boolean"
+        },
+        "tcpcSupport": {
+          "description": "whether type-C port controller support is built in",
+          "type": "boolean"
+        },
+        "usbTypecRevision": {
+          "description": "usb type-c spec revision supported by a device",
+          "examples": [
+            "1.2",
+            "1.3"
+          ],
+          "type": "string"
+        },
+        "pdVersion": {
+          "description": "version of power delivery spec supported by a device",
+          "examples": [
+            "pd2.0",
+            "pd3.1"
+          ],
+          "type": "string"
+        },
+        "pins": {
+          "description": "array of pin objects with associated properties",
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/pinSpec"
+          }
+        },
+        "package": {
+          "description": "component's package size and description",
+          "items": {
+            "$ref": "definitions.json#/package"
+          }
```

Comment:
not "items", just:

"$ref": "definitions.json#/package"

Reply:
changed and also made the change for other components where package is called,